### PR TITLE
Fix #1789 - Level 200 summon doesn't follow up and down stairs

### LINKED ISSUE #1329: Pack de Errors - Summon (PET)
1- Você apanha de  seu próprio summon.
image

2- summon indo parar dentro de água.
image

3- ao relogar, summon não desaparece.

4- todas magias de attack estão com effects / dano /  area, totalmente errados.

image

### LINKED ISSUE #1789: Level 200 summon doesn't follow up and down stairs and doesn't use aoe abilities or heal
**Describe the bug**
Level 200 summon doesn't follow up and down stairs anymore
also isnt using all abilities

**To Reproduce**
1. Summon Emberwing
2. Go up or down stairs

**Expected behavior**
Summon should teleport to owner

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [Ubuntu 20.04]

### PR DESCRIPTION
**Pull request by heberholanda**

### CODE REVIEW

File: src/creature.cpp [hunk 1]
```diff
@@ -468,11 +468,9 @@ void Creature::onCreatureMove(Creature* creature, const Tile* newTile, const Pos
 			std::forward_list<Creature*> despawnList;
 			for (Creature* summon : summons) {
 				const Position& pos = summon->getPosition();
-				if (summon->getMonster()->isPet() && (Position::getDistanceZ(newPos, pos) >= 1 || (std::max<int32_t>(Position::getDistanceX(newPos, pos), Position::getDistanceY(newPos, pos)) >= 15))) {
-					g_game.internalTeleport(summon, summon->getMaster()->getPosition(), true);
-				} else if (!summon->getMonster()->isPet() && (Position::getDistanceZ(newPos, pos) >= 2 || (std::max<int32_t>(Position::getDistanceX(newPos, pos), Position::getDistanceY(newPos, pos)) >= 30))) {
-					despawnList.push_front(summon);
-				}
+        if (Position::getDistanceZ(newPos, pos) > 0 || (std::max<int32_t>(Position::getDistanceX(newPos, pos), Position::getDistanceY(newPos, pos)) > 10)) {
```
Comment (lgrossi) [inline src/creature.cpp]: Could you please elaborate on those magic numbers? 
If I understood well, you want it to teleport when in different floors or whenever it leaves the screen, what is fine.
Im quite sure we have variables to define screen size, that would make it a little bit more clear.

**Review by lgrossi — COMMENTED** [review]
Maybe clarify a bit the magic numbers.

**Comment by debone** [comment]
Tested. Maybe we want the distance back to 15? It was very easy to make the summons jump.

**Comment by heberholanda** [comment]
> Tested. Maybe we want the distance back to 15? It was very easy to make the summons jump.

ok, I set 15 now.

**Review by debone — APPROVED** [review]

**Pull request merged**